test/Microsoft.ML.FSharp.Tests/Microsoft.ML.FSharp.Tests.fsproj

```diff
@@ -4,7 +4,6 @@
     <!-- Empty TargetFramework because test\Directory.Build.props sets TFM, but FSharp.Tests is multi-targeted -->
```

Comment:
Actually, this whole chunk can be removed now. F# tests will get netfx coverage through the netfx leg. There is no reason for this project to be multi-targeted.

Remove this comment and the 2 lines below:

```
    <TargetFramework></TargetFramework>
    <TargetFrameworks>$(DefaultTestTargetFramework)</TargetFrameworks>
```